src/app/UI/widgets/card/card.component.scss

```
@@ -0,0 +1,95 @@
+  $background: #D8E3FA;
+  $gray: #EAEAEC;
+  $primary: #29283F;
+  $primary-violet: #7370F4;
+
+
+body {
+  background-color: $background;
+}
+
+* {
+  margin: 0px;
+  padding: 0px;
+}
```

```suggestion
body {
  background-color: $background;
}

* {
  margin: 0;
  padding: 0;
}
```

This section should be added on `src/styles.scss` also, when you have `0px` attributes, the notation should be just `0`.